pyproject.toml

```diff
@@ -192,3 +196,19 @@ strict = true
 [tool.codespell]
 skip = ".git,*.pdf,*.svg"
 ignore-words-list = "theis,coo,homogenous"
+
+[tool.towncrier]
+package = "anndata"
+directory = "docs/release-notes"
+filename = "docs/release-notes/{version}.md"
+single_file = false
+package_dir = "src"
+issue_format = "[#{issue}](https://github.com/scverse/anndata/pull/{issue})"
```

Comment:
```suggestion
issue_format = "{{issue}}`{issue}`"
```

Reply:
Why? This is the current format that we have

Reply:
The syntax we use in our docs is ``{issue}`123` ``

Is this not used in our docs?

Reply:
Ah, I see what this would do.  Apologies